src/fieldValidators.js

```diff
@@ -78,6 +78,18 @@ const testEmailFormat = (value: ?string) => {
     return regexp.test(value.trim()) && !regexpAccent.test(value.trim()) ? null : 'notEmail';
 };
 
+/**
+ * @desc Test if a value is a valid linkedin url
+ * @param {?string} value - The value of the field
+ * @return {?string} 'notLinkedinUrl' if the value is not a valid linkedin Url, null if it is
+ */
+const testLinkedinUrlFormat = (value: ?string) => {
```

Comment:
On n'écrit pas du `LinkedIn` plutôt que `Linkedin` d'habitude ?

Reply:
Yes I fix that